src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -1339,24 +1338,36 @@ public void TestNonBlockingDatabaseOperationConflictResolver()
                 if (resolveCnt == 0) {
                     using (var d = Db.GetDocument("doc1"))
                     using (var doc = d.ToMutable()) {
-                        d.GetString("name").Should().Be("Tiger");
+                        d.GetString("name").Should().Be("Cat");
                         doc.SetString("name", "Cougar");
                         Db.Save(doc);
-                        d.GetString("name").Should().Be("Cougar", "Because database save operation was not blocked");
+                        doc.GetString("name").Should().Be("Cougar", "Because database save operation was not blocked");
```

Comment:
Not a problem with this PR in particular, but while you are fixing this test, this check doesn't actually check a proper save, it just checks that the in memory value was updated.  After the save, retrieve the document again and check the string to get a proper validation.